draft-ietf-gnap-core-protocol.md

```diff
@@ -125,37 +125,27 @@ Roles are defined by the actions taken and the expectations leveraged
 on the role by the overall protocol. 
 
 Authorization Server (AS)
-: Manages the requested delegations for the RO. 
-    The AS issues tokens and directly delegated information to an instance of the client.
-    The AS is defined by its grant endpoint, a single URL that accepts a POST
-    request with a JSON payload. The AS could also have other endpoints,
-    including interaction endpoints and user code endpoints, and these are
-    introduced to the RC as needed during the delegation process. 
+: server that grants privileges to a particular end-user and that provides them to a client in the form of an access token. 
 
 Client
-: Requests tokens and directly delegated information from the AS, and uses tokens at the RS.
-    For some kinds of client software, there could be many instances
-    of a single piece of client software. This specification
-    differentiates between a specific instance (the client instance) and the 
-    software running the instance (the client software).
-    A client instance is identified by its unique key, which can
-    be known to the AS prior to the first request or introduced
-    to the AS as part of the protocol. The AS determines
-    which policies apply to a given client instance, including what it can
-    request and on whose behalf. 
-
-Resource Server (RS, aka "API")
-: Accepts tokens from the client instance issued by the AS and serves delegated resources
-    on behalf of the RO. There could be multiple RSs protected
-    by the AS that the client instance will call.
+: application operated by an end-user that consumes resources from one or several RSs, possibly requiring access privileges from one or several ASs. 
+
+    Note: this specification differentiates between a specific instance (the Client instance, identified by its public key) and the software running the instance (the Client software). For some kinds of Client software, there could be many instances of a single piece of Client software.
+
+    Example: a Client can be a mobile application, a web application, etc.
+
+Resource Server (RS)
+: server that provides operations on resources where protected operations require that the client provides one or more valid access tokens issued by one or more ASs.
```

Comment:
Suggest we remove "client" from this as the RS isn't concerned with (and often doesn't know about) the client software, just the token. I know there was some discussion on this but I also don't care for the "one or more" items here. By saying "a token" and "an AS" we aren't limiting the RS to also accept others, but it's defined by accepting at least one.

```suggestion
: server that provides operations on resources where protected operations require a valid access token issued by an AS trusted by the RS for this purpose.
```

Reply:
There was the question whether the definitions should allow the possibility for several servers. If we want to reflect the current state of affairs, your proposal is better.

Reply:
My proposal :allows: several servers without, in my opinion, confusing the matter.

Reply:
ok with that.

Reply:
I didn't include "trusted by the RS for this purpose." because I don't want a circular definition